### PR TITLE
Apply example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Level 0*

## Solution
- *apply example_ynh*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** :
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/mediawiki_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Review by Salamandar — COMMENTED** [review]
Can you please remove the changes of the conf/*.src files ? I already commited the updates of the urls.

LGTM apart from the fact that you removed the translations of the manifest.json.

**Comment by Salamandar** [comment]
Oh, you also removed is_public=$(…) in the upgrade script (CI failure).

**Comment by yalh76** [comment]
> 
> 
> Can you please remove the changes of the conf/*.src files ? I already commited the updates of the urls.
> 
> LGTM apart from that..

Just merged master to example was enough to solve that

**Comment by yalh76** [comment]
> 
> 
> Oh, you also removed is_public=$(…) in the upgrade script (CI failure).

I've seen. it's because `is_public will no more be used`

**Comment by yalh76** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -16,128 +16,150 @@ ynh_script_progression --message="Loading installation settings..." --weight=1
...
 
 # Fix is_public as a boolean value
 if [ "$is_public" = "Yes" ]; then
-    ynh_app_setting_set --app="$app" --key=is_public --value=1
+    ynh_app_setting_set --app=$app --key=is_public --value=1
     is_public=1
 elif [ "$is_public" = "No" ]; then
-    ynh_app_setting_set --app="$app" --key=is_public --value=0
+    ynh_app_setting_set --app=$app --key=is_public --value=0
     is_public=0
```
Comment (Salamandar) [inline scripts/upgrade:69]: So I guess this could be removed ? Is it not in the example_ynh anymore.

Comment (yalh76) [reply scripts/upgrade:69]: Yes It should be remove if we want to use new system permission

**Pull request merged**